notebooks/bigQuery.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4d79e85b",
+   "metadata": {},
+   "source": [
+    "## Big Query SQL queries used in this project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df71635a",
+   "metadata": {},
+   "source": [
+    "Here we're collecting lists of page paths and content ids, for which there have been more than 5 page hits in the last 2 weeks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dc65c551",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import packages\n",
+    "import pandas as pd\n",
+    "import pandas_gbq\n",
+    "\n",
+    "# extract data from BigQuery\n",
+    "project_id = \"govuk-bigquery-analytics\"\n",
+    "\n",
+    "# document types to ignore i.e. pages not relevant to WUJs\n",
+    "ignore = (\n",
+    "    \"authored_article\",\n",
+    "    \"news_article\",\n",
+    "    \"news_story\",\n",
+    "    \"press_release\",\n",
+    "    \"world_news_story\",\n",
+    "    \"utaac_decision\",\n",
+    "    \"speech\",\n",
+    "    \"case_study\",\n",
+    "    \"raib_report\",\n",
+    "    \"asylum_support_decision\",\n",
+    "    \"policy_paper\",\n",
+    "    \"corporate_report\",\n",
+    "    \"written_statement\",\n",
+    "    \"consultation_outcome\",\n",
+    "    \"closed_consultation\",\n",
+    "    \"maib_report\",\n",
+    "    \"person\",\n",
+    "    \"correspondence\",\n",
+    "    \"employment_tribunal_decision\",\n",
+    "    \"employment_appeal_tribunal_decision\",\n",
+    "    \"tax_tribunal_decision\",\n",
+    "    \"ministerial_role\",\n",
+    "    \"residential_property_tribunal_decision\",\n",
+    "    \"cma_case\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9a07dc8e",
+   "metadata": {},
+   "source": [
+    "### By page hits per page path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "187147c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get page paths with more than 5 page hits in the last 2 weeks,\n",
+    "# ignoring irrelevant pages\n",
+    "sql = \"\"\"\n",
+    "DECLARE start_date STRING DEFAULT \"20211018\";\n",
```

Comment:
For a two week period, so we don't duplicate the same days (as the day impacts which pages are visited on gov.uk), a two-week period would be 20211018 - 20211031.